### ISSUE TITLE
Pod coverage tests may fail

### ISSUE THREAD

**Issue by eserte** [neilneely/Data-Serializer#4]
t/10-02-Pod-Coverage.t fails on most of my smoker systems --- probably this happens if optional prereqs are not installed:
```
#   Failed test 'Pod coverage on Data::Serializer::Bencode'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::Bencode: requiring 'Data::Serializer::Bencode' failed

#   Failed test 'Pod coverage on Data::Serializer::FreezeThaw'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::FreezeThaw: requiring 'Data::Serializer::FreezeThaw' failed

#   Failed test 'Pod coverage on Data::Serializer::Convert::Bencode'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::Convert::Bencode: requiring 'Data::Serializer::Convert::Bencode' failed

#   Failed test 'Pod coverage on Data::Serializer::Convert::Bencode_XS'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::Convert::Bencode_XS: requiring 'Data::Serializer::Convert::Bencode_XS' failed

#   Failed test 'Pod coverage on Data::Serializer::Data::Denter'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::Data::Denter: requiring 'Data::Serializer::Data::Denter' failed

#   Failed test 'Pod coverage on Data::Serializer::Data::Taxi'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::Data::Taxi: requiring 'Data::Serializer::Data::Taxi' failed

#   Failed test 'Pod coverage on Data::Serializer::PHP::Serialization'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::PHP::Serialization: requiring 'Data::Serializer::PHP::Serialization' failed

#   Failed test 'Pod coverage on Data::Serializer::XML::Dumper'
#   at /usr/perl5.28.2p/lib/site_perl/5.28.2/Test/Pod/Coverage.pm line 133.
# Data::Serializer::XML::Dumper: requiring 'Data::Serializer::XML::Dumper' failed
# Looks like you failed 8 tests of 20.
t/10-02-Pod-Coverage.t ........ 
Dubious, test returned 8 (wstat 2048, 0x800)
Failed 8/20 subtests 
```

**Comment by neilneely** [comment]
For now I've taken the easy route of trimming down the pod test to just the base Data::Serializer module.  Should be fixed with 0.64 that was uploaded a few minutes ago.

**Comment by eserte** [comment]
Actually it was the pod coverage test which was failing, not the pod test.

**Comment by neilneely** [comment]
Sorry, my mistake.  Fixed correctly in 0.65 - and is looking much better now.
